pkg/ctl/cmdutils/filter/nodegroup_filter_test.go

```diff
@@ -334,317 +334,317 @@ func addGroupB(cfg *api.ClusterConfig) {
 }
 
 const expected = `
-  {
```

Comment:
Why the formatting change here?

Reply:
Ah, that's sort of left over from the original version of this PR where I had updated the cluster name in this expected data and then fixed the mixed indentation. Now that the cluster name hasn't been changed, it's just the fixing of the mixed (and over/under) indentation. I figured it was fine to leave it in for codebase health, but I can pull it out since it's not coupled to any other changes in this PR.